src/Controller/SiteController.php

```diff
@@ -33,6 +52,55 @@ public function index(): ResponseInterface
         return $response;
     }
 
+    public function contact(RequestInterface $request): ResponseInterface
+    {
+        $parameters = [];
+        if ($request->getMethod() === 'POST') {
+            $config = array_merge(
+                require $this->aliases->get('@root/config/params.php'),
+                require $this->aliases->get('@root/config/params-local.php'),
+            );
+            $sent = false;
+            $error = '';
+            try {
+                foreach (['subject', 'name', 'email', 'content'] as $name) {
+                    if (empty($_POST[$name])) {
```

Comment:
Shouldn't we be using PSR7 request object features instead?

Reply:
Yes, we should.